validation/src/test/scala/hmda/validation/rules/ts/validity/ValidityUtils.scala

```diff
@@ -0,0 +1,24 @@
+package hmda.validation.rules.ts.validity
+
+import hmda.model.fi.ts.Respondent
+
+trait ValidityUtils {
+  def respondentNotEmpty(r: Respondent): Boolean = {
```

Comment:
Based on the name, I'd expect this method to be the opposite (that is, the complement) of `respondentIsEmpty`.  In other words, I'm surprised that a Respondent with a name but no address (for example) would have both these functions return false: it's neither empty nor not-empty.  I'm not sure what names would make it clear: empty and full, perhaps?  (Or perhaps empty and not-empty are domain terms that mean exactly what they mean here, in which case, a comment might be in order.)

Also, I don't think all the cases of `V105` are covered, which feels somewhat related.  For example, I can change its first `and` to `or` and still have all the tests pass. (Not part of this PR, perhaps, but the code is here so I'm commenting here.)